snuba/migrations/parse_schema.py

```diff
@@ -46,7 +46,7 @@
     enum_val         = ~r"\d+"
     agg              = "AggregateFunction" open_paren space* agg_func space* comma space* agg_types space* close_paren
     agg_func         = ~r"[a-zA-Z0-9]+\([a-zA-Z0-9\,\.\s]+\)|[a-zA-Z0-9]+"
-    agg_types        = (primitive (space* comma space*)?)*
+    agg_types        = (type (space* comma space*)?)*
```

Comment:
Why does this need to change?

Reply:
The `Nullable()` wrapping.

Reply:
That would have been preferred.  At the time I didn't know how to accomplish that.  I'm guessing in the materialized view I'd do something like `if(isNull(column), a, b)`?  I don't really want to back fill for another 90 days.  Do you have familiarity with migrations from nullable to not nullable and the performance implications?

Reply:
>  I'm guessing in the materialized view I'd do something like if(isNull(column), a, b)?

Correct

> I don't really want to back fill for another 90 days. Do you have familiarity with migrations from nullable to not nullable and the performance implications?

That makes sense to me. I didn't know that the aggregated table is already being written to for such a long time. You won't be able to change a nullable column to non-nullable unfortunately. The only way I can think of is to  create new columns in aggregate tables which are non-nullable. And in the materialization write to both the old and new columns. And when you have both you can update the query processors to add a column mapper which maps old columns to new columns. But this is a heavy process. Might as well go with what you have now and revisit if you still have problems.